src/app/backend/resource/container/logs_test.go

```diff
@@ -47,6 +48,40 @@ var log5 = logs.LogLine{
 	Content:   "log5",
 }
 
+var details *logs.LogDetails
+
+func benchmarkGetLogDetails(lines int, timestamp string, b *testing.B) {
```

Comment:
IIRC benchmarks are not executed as part of the test spec by default (they need `-bench` arg). It's a nice addition though. Have you compared old implementation performance of `getLineIndex` with the new one?

Reply:
@floreks Yes I compared the performance. The summary is here: https://github.com/kubernetes/dashboard/issues/3900#issuecomment-517395365.

Overall, there's a negligible speedup.